modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -11,52 +20,116 @@ If you do not need to review the JSON files that the `ccoctl` tool creates befor
+//Note: those Alibaba Cloud docs links are officially intended for user consumption (https://github.com/openshift/cloud-credential-operator/pull/412/files#r764509590). We would prefer not to link to GitHub like this, but an alternative has not yet presented itself.
+endif::alibabacloud[]
 
 .Procedure
 
 . Extract the list of `CredentialsRequest` objects from the {product-title} release image:
 +
 [source,terminal,subs="+quotes"]
+ifdef::aws[]
 ----
-$ oc adm release extract --credentials-requests --cloud=aws --to=__<path_to_directory_with_list_of_credentials_requests>__/credrequests quay.io/__<path_to>__/ocp-release:__<version>__
+$ oc adm release extract --credentials-requests --cloud=aws --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
 ----
+endif::aws[]
+ifdef::alibabacloud[]
+----
+$ oc adm release extract --credentials-requests --cloud=alibabacloud --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/ocp-release:<version>
+----
+endif::alibabacloud[]
 
 . Use the `ccoctl` tool to process all `CredentialsRequest` objects in the `credrequests` directory:
 +
+ifdef::aws[]
 [source,terminal,subs="+quotes"]
 ----
-$ ccoctl aws create-all --name=__<name>__ --region=__<aws_region>__ --credentials-requests-dir=__<path_to_directory_with_list_of_credentials_requests>__/credrequests
+$ ccoctl aws create-all --name=<name> --region=<aws_region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests
 ----
++
+where:
++
+** `<name>` is the name used to tag any cloud resources that are created for tracking.
+** `<aws-region>` is the AWS region in which cloud resources will be created.
+** `<path_to_directory_with_list_of_credentials_requests>/credrequests` is the directory containing the files for the component `CredentialsRequest` objects.
+endif::aws[]
+ifdef::alibabacloud[]
+[source,terminal,subs="+quotes"]
+----
+$ ccoctl alibabacloud create-ram-users --name <name> --region=<alibaba-region> --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests --output-dir=<path_to_ccoctl_output_dir>/manifests
+----
++
+where:
++
+** `<name>` is the name used to tag any cloud resources that are created for tracking.
+** `<alibaba-region>` is the Alibaba Cloud region in which cloud resources will be created.
+** `<path_to_directory_with_list_of_credentials_requests>/credrequests` is the directory containing the files for the component `CredentialsRequest` objects.
+** `<path_to_ccoctl_output_dir>/manifests` is the directory where the component credentials secrets are generated.
```

Comment:
Same here. No need to specify `manifests` as part of the path, as it will be created underneath path_to_ccoctl_ouptput_dir